src/common/components/hosts/utils.ts

```diff
@@ -71,6 +67,8 @@ export const canEditHost = (clusterStatus: Cluster['status'], status: Host['stat
     'pending-for-input',
   ].includes(status);
 
+export const canEditRole = canEditHost;
```

Comment:
Does it work fine with SNO? I just realised that we'll need to also keep the `isSNO` check
```suggestion
export const canEditRole = (clusterStatus: Cluster['status'], status: Host['status'], isSNO?: boolean) => canEditHost(clusterStatus, status) && !isSNO;
```

Reply:
It's true. I'll change it